lib/src/screens/supa_email_auth.dart

```diff
@@ -70,93 +70,87 @@ class _SupaEmailAuthState extends State<SupaEmailAuth> {
-                signingIn ? 'Sign In' : 'Sign Up',
-                style: const TextStyle(fontWeight: FontWeight.bold),
-              ),
-              onPressed: () async {
-                if (_formKey.currentState!.validate() && signingIn) {
-                  final res = await supaAuth.signInExistingUser(
-                      _email.text, _password.text);
-                  if (res.error?.message != null) {
-                    await showDialog(
-                      context: context,
-                      builder: (BuildContext context) {
-                        return AlertDialog(
-                          title: Text(res.error!.message),
-                          contentTextStyle: const TextStyle(
-                            backgroundColor: Colors.red,
-                          ),
-                        );
-                      },
-                    );
-                    _email.text = '';
-                    _password.text = '';
-                  } else {
-                    await showDialog(
-                      context: context,
-                      builder: (BuildContext context) {
-                        return const AlertDialog(
-                          title: Text('Success!'),
-                          contentTextStyle: TextStyle(
-                            backgroundColor: Colors.green,
-                          ),
-                        );
-                      },
-                    );
-                    if (!mounted) return;
-                    Navigator.popAndPushNamed(
-                        context, widget.redirectUrl ?? '');
-                  }
-                } else {
-                  final res = await supaAuth.createNewEmailUser(
-                      _email.text, _password.text);
-                  if (res.error?.message != null) {
-                    await showDialog(
-                      context: context,
-                      builder: (BuildContext context) {
-                        return AlertDialog(
-                          title: Text(res.error!.message),
-                          contentTextStyle: const TextStyle(
-                            backgroundColor: Colors.red,
-                          ),
-                        );
-                      },
-                    );
-                  } else {
-                    await showDialog(
-                      context: context,
-                      builder: (BuildContext context) {
-                        return const AlertDialog(
-                          title: Text('Success!'),
-                          contentTextStyle: TextStyle(
-                            backgroundColor: Colors.red,
-                          ),
-                        );
-                      },
-                    );
-                    if (!mounted) return;
-                    Navigator.popAndPushNamed(
-                        context, widget.redirectUrl ?? '');
-                  }
+          ElevatedButton(
```

Comment:
I have removed the surrounding SizedBox and used `crossAxisAlignment: stretch` on the parent column.